README.md

```diff
@@ -24,7 +24,7 @@ Tastier than ever!
     1. Make sure you have a build environment setup.
     2. Make a new directory, cd to it and run
         ```
-        repo init -u https://github.com/PotatoProject/manifest -b croquette-release;
+        repo init -u --depth=1 https://github.com/PotatoProject-next/manifest -b d-staging1;
```

Comment:
Let's not default to depth to 1, rest looks good :)

Reply:
Thanks